packages/openactive-broker-microservice/src/util/item-transforms.js

```diff
@@ -0,0 +1,91 @@
+const { partition, omit } = require('lodash');
+
+/**
+ * Inverts any items that have an `individualFacilityUse` property, so that the top-level `kind` is "IndividualFacilityUse"
+ *
+ * @param {{state: string, modified:any, kind: string, id: string, data: {individualFacilityUse: object}}[]} items
+ */
+function invertFacilityUseItems(items) {
+  const [invertibleFacilityUseItems, otherItems] = partition(items, (item) => item.data?.individualFacilityUse?.length > 0);
+  if (invertibleFacilityUseItems.length < 1) return items;
+
+  // Invert "FacilityUse" items so the the top-level `kind` is "IndividualFacilityUse"
+  const invertedItems = [];
+  for (const facilityUseItem of invertibleFacilityUseItems) {
+    for (const individualFacilityUse of facilityUseItem.data.individualFacilityUse) {
+      invertedItems.push({
+        ...facilityUseItem,
+        kind: individualFacilityUse['@type'],
+        id: individualFacilityUse['@id'],
+        data: {
+          ...individualFacilityUse,
+          '@context': facilityUseItem.data['@context'],
+          aggregateFacilityUse: omit(facilityUseItem.data, ['individualFacilityUse', '@context']),
+        },
+      });
+    }
+  }
+
+  return invertedItems.concat(otherItems);
+}
+
+/**
+ * Creates an opportunity item from a subEvent. This is used when a SessionSeries feed has ScheduledSessions,
+ * and using this function splits the subEvents into items to simulate a ScheduledSessions feed.
```

Comment:
```suggestion
 * Creates an opportunity item from a subEvent. This is used when a SessionSeries feed has embedded ScheduledSessions,
 * and using this function splits the subEvent into items to simulate a separate ScheduledSessions feed.
```

"embedded" and "separate" to borrow the parlance from https://developer.openactive.io/publishing-data/data-feeds/types-of-feed